c/public/lib/source/ttread/ttread.c

```diff
@@ -2426,7 +3242,7 @@ static void callbackApproxPath(ttrCtx h, GID gid, abfGlyphCallbacks *glyph_cb) {
 static void readGlyph(ttrCtx h,
                       unsigned short gid, abfGlyphCallbacks *glyph_cb) {
     int result;
-    long nContours;
+    int nContours;
```

Comment:
Please initialize `nContours` to zero, as Xcode's static analysis thinks it can be used before initialization below.

Reply:
initialized